app/serializers/search/listing_serializer.rb

```diff
@@ -6,6 +6,7 @@ class ListingSerializer < ApplicationSerializer
                :category,
                :contact_via_connect,
                :expires_at,
+               :created_at,
```

Comment:
Listing API was not passing created_at date now so I added it.

Reply:
This API  https://dev.to/search/listings?category=&listing_search=&page=0&per_page=75&tag_boolean_mode=all